### ISSUE TITLE
Check mark icon not aligned in center

### ISSUE THREAD

**Issue by piyush0506** [vonovak/react-native-round-checkbox#10]
I'm using the example code something like this:
```
<RoundCheckbox
  size={24}
  checked={this.state.isSelected}
  onValueChange={(newValue) => {console.log(newValue)}}
/>
```
It looks something like this:
![Screenshot_20200812_184423](https://user-images.githubusercontent.com/30774661/90019479-1a3f0c00-dccc-11ea-9332-2a2ece939c5f.jpg)

Icon is not aligned in center

Using following dependancies:

```
"react": "16.13.1",
"react-native": "0.63.1",
"rn-round-checkbox": "^1.0.0",
"react-native-vector-icons": "^7.0.0",

```
Anyone else having the solution?

**Comment by naheed-shamim** [comment]
Just go to the implementation file and update this


const iconSize = parseInt(size * 1.1, 10);   <------- UPDATE HERE, Change this to  -> **const iconSize = size;**

**Comment by piyush0506** [comment]
Yes it works, but it's not permanent solution, once we run npm install / yarn install, our changes would be vanished.
Looking for a permanent solution or any prop in it

**Comment by lorenjohnson** [comment]
+1

**Comment by Vickout** [comment]
Multiply the size by 1.1 does the icon greater than the checkbox itself. If it was smaller like 0.75 it would be more suitable.

**Comment by diatche** [comment]
Yan can use [patch-package](https://github.com/ds300/patch-package) to make the changes in the implementation file permanent.

**Comment by MiracleUFO** [comment]
Why can't this be fixed permanently in the package?
